internal/ai/analyze_test.go

```
@@ -0,0 +1,227 @@
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+// Helper functions for environment variable management in tests
+func getEnvOrEmpty(key string) string {
```

golangci-lint (1.64.8)

func `getEnvOrEmpty` is unused

(unused)